docs-source/content/security/openshift.md

```diff
@@ -5,14 +5,111 @@ weight: 7
+
+Here is an excerpt from a standard WebLogic [Dockerfile](https://github.com/oracle/docker-images/blob/master/OracleWebLogic/dockerfiles/12.2.1.4/Dockerfile.generic#L89)
+that demonstrates how the file system group ownership is configured in the standard WebLogic images:
+
+```bash
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
+# ------------------------------------------------------------
+RUN mkdir -p /u01 && \
+    chmod 775 /u01 && \
+    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle && \
+    chown oracle:root /u01
+
+COPY --from=builder --chown=oracle:root /u01 /u01
+```
+
+However, if you build your own image, or obtain an image from another source, it
+may not have the necessary permissions.  You may need to configure similar file
+system permissions to allow your image to work in OpenShift.
+
+Alternatively, you may choose to configure OpenShift to allow use of UID 1000.  This
+can be done using a Security Context Constraint.  Oracle recommends that you define
+a custom Security Context Constraint that has just the permissions that are required
+and apply that to WebLogic pods.  If you prefer, you can use the built-in `anyuid`
+Security Context Constraint, but you should be aware that it provides more permissions
+than are needed, and is therefore a less secure option.
+
+#### Create a custom Security Context Constraint
+
+To create a custom security context constraint, create a YAML file with the following
+content.  This example assumes that your OpenShift project is called `weblogic` and
+that the service account you will use to run the WebLogic Operator and domains
+is called `weblogic-operator`.  You should change these
+in the `groups` and `users` sections to match your environment.
+
+```yaml
+kind: SecurityContextConstraints
+apiVersion: v1
+metadata:
+  name: uid1000
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: false
+fsGroup:
+  type: MustRunAs
+groups:
+- system:serviceaccounts:weblogic
+readOnlyRootFilesystem: false
+requiredDropCapabilities:
+- KILL
+- MKNOD
+- SETUID
+- SETGID
+runAsUser:
+  type: MustRunAs
+  uid: 1000
+seLinuxContext:
+  type: MustRunAs
+supplementalGroups:
+  type: RunAsAny
+users:
+- system:serviceaccount:weblogic:weblogic-operator
+volumes:
+- configMap
+- downwardAPI
+- emptyDir
+- persistentVolumeClaim
+- projected
+- secret
+```
+
+Assuming you called that file `uid1000.yaml`, you can create the Security Context Constraint
+using the following command:
+
+```bash
+$ oc create -f uid1000.yaml
+```
+
+After you have created the security context constraint, you can install the WebLogic Operator.
+Make sure you use the same service account that you granted permission to in the security
+context constraint (`weblogic-operator` in the example above).  The operator will then run
```

Comment:
service account that you granted permission to -> service account to which you granted permission
the example above -> the preceding example